src/js/components/sidebar.jsx

```diff
@@ -97,37 +100,67 @@ export default class SideBar extends React.Component {
       filter,
       filteredTabs,
       selectAll,
+      tabIds,
       tabsById,
       moreActionsShown,
     } = this.state;
 
     const tabs = filteredTabs.map(tabId => {
       const tabInfo = tabsById.get(tabId);
+      const tabFilteredIndex = filteredTabs.indexOf(tabId);
+
+      let pinMarginEnabled = false;
+
+      if (tabFilteredIndex + 1 < filteredTabs.length) {
+        const nextTab = tabsById.get(filteredTabs[tabFilteredIndex + 1]);
+        if (tabInfo.pinned && !nextTab.pinned) {
+          pinMarginEnabled = true;
```

Comment:
I feel like this pinMarginEnabled stuff can probably be done a different way. CSS to the rescue!

What I suggest is that we give each pinned TabInfo's root `<li>` a `pinned` attribute, and then add the following CSS rule:

    li[pinned] + li:not([pinned]) {
      margin-top: 10px;
    }

Which should add the spacing we want between the last pinned tab and the first non-pinned tab.

Reply:
Great idea 👍 ! Since I already use a class `pinned` for pinned tabs, I prefere using the class selector instead of using another attribute named `pinned.`

I implemented the following CSS rule:

`
li.pinned + li:not(.pinned) {
   margin-top: 10px;
}
`